cibuildwheel/architecture.py

```diff
@@ -74,20 +74,14 @@ def parse_config(config: str, platform: PlatformName) -> set[Architecture]:
     def auto_archs(platform: PlatformName) -> set[Architecture]:
         native_machine = platform_module.machine()
 
-        # Cross-platform support. Used for --print-build-identifiers or docker builds.
-        host_platform: PlatformName = (
-            "windows"
-            if sys.platform.startswith("win")
-            else ("macos" if sys.platform.startswith("darwin") else "linux")
-        )
```

Comment:
I made this PR because it was bothering me that the logic here does not exactly match the logic in `_compute_platform_ci()`.